src/net/sourceforge/kolmafia/textui/command/MayamCommand.java

```diff
@@ -0,0 +1,103 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.session.InventoryManager;
+
+public class MayamCommand extends AbstractCommand {
+  private static final List<List<String>> SYMBOL_POSITIONS =
+      List.of(
+          List.of("yam", "clock", "explosion"),
+          List.of("yam", "eyepatch", "cheese", "wall"),
+          List.of("yam", "lightning", "bottle", "board", "meat"),
+          List.of("yam", "sword", "vessel", "fur", "chair", "eye"));
+
+  public MayamCommand() {
+    var symbols =
```

Comment:
might want to reverse the list, because you have to input the symbols largest ring to smallest, but they're displayed smallest ring to largest.

Reply:
I want to do it such that ‘eye yam eyepatch yam’ would go in that order

Reply:
the order is fine, it's the help text that starts "yam|clock|explosion", but I'm saying explosion should be one of the last instead of one of the first